src/components/DefaultModal.tsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import styled from 'styled-components';
+
+export const Overlay = styled.div`
+	position: absolute;
+	backdrop-filter: blur(5px);
+	height: 100%;
+	width: 100%;
+	display: flex;
+	// justify-content: center;
+	align-items: center;
+
+	&.closed {
+		display: none;
+	}
+`;
```

Comment:
I am starting to understand how the className isOpen works, but this isn't quite how we want to do it.

I think this is a bit more code, but I think the user of the Default modal just has to think about the modal as open true or false, not implementation details of how that is done in the DefaultModal.


```suggestion
export const Overlay = styled.div<{open: boolean}>`
	position: absolute;
	backdrop-filter: blur(5px);
	height: 100%;
	width: 100%;
	display: flex;
	// justify-content: center;
	align-items: center;

        ${({ open }) =>
            !props.open &&
            css`
                display: none;
            `}
`;
```

Reply:
Never realized I could do this! This seems much better, but I don't think the `props` of `props.open` or the `css` is necessary?

Reply:
@BraidenCutforth @isaiahdoyle Another way to do this to avoid putting logic in the CSS would be to do something like
```js
className={isOpen ? 'open' : 'closed'}
```
And have two css classes: `.open {}` and `.closed {}`

Reply:
I think really what we can do is conditional rendering higher up the component tree. I made a large review comment about that.